docs/experiments-gov2.md

```diff
@@ -31,15 +31,17 @@ eval/trec_eval.9.0/trec_eval src/main/resources/topics-and-qrels/qrels.701-750.t
 You should be able to replicate the following results:
 
 MAP                                                                                     | BM25   |BM25+RM3| QL     | QL+RM3
-:---------------------------------------------------------------------------------------|--------|--------|--------|--------
-[TREC 2004 Terabyte Track: Topics 701-750](http://trec.nist.gov/data/terabyte04.html)   | 0.2673 | 0.2953 | 0.2635 | 0.2800
+----------------------------------------------------------------------------------------|--------|--------|--------|--------
+[TREC 2004 Terabyte Track: Topics 701-750](http://trec.nist.gov/data/terabyte04.html)   | 0.2673 | 0.2953 | 0.2636 | 0.2800
```

Comment:
I am fine to keep it as is ... since the numbers are pretty much the same...
But I am not sure whether the difference comes from indexing...
`trec_eval` sorts the documents using the scores no matter the orders in the original ranking list. It breaks the ties by alphabetical order.
That means there should be some difference in the ranking list...